t/features/custom-ops.t

```diff
@@ -504,4 +504,20 @@ use _007::Test;
     outputs $program, "James\n", "can declare an operator with a backslash in the name";
 }
 
+{
+    my $program = q:to/./;
+        sub postfix:<!>(x) is looser(prefix:<^>) {
```

Comment:
Hm, may I suggest while #190 is open that we don't create operators `postfix:<!>` &mdash; they might make tests fail for the wrong reason.